Jenkinsfile

```
@@ -573,6 +427,7 @@ pipeline {
                     steps {
                         sconsBuild clean: "_build.external${arch}",
                                    parallel_build: parallel_build(),
+                                   log_to_file: 'centos7-gcc-build.log',
```

For future matrix-ing flexibility, this file doesn't need to be named for the stage does it?  We don't archive it or anything so it's isolated to this stage.
```suggestion
                                   log_to_file: 'build.log',
```

---

That was just tossed in to get around the problem that recordIssues was having.

I was just being consistent with the naming.

By giving it a unique name, if we have to resort to looking in a build agent's workspace for debugging a PR, it makes it easier to find.

The developers may decide that the recordIssues and this log file is not needed based on some of the conversations.

Until then, I probably need to add log files to the other build steps for the same reason I added one here.